ui/espresso/IdlingResourceSample/build.gradle

```diff
@@ -44,5 +44,5 @@ ext {
     coreVersion = "1.1.0-beta01"
     extJUnitVersion = "1.1.0-beta01"
     runnerVersion = "1.1.1-beta01"
```

Comment:
I assumed master would be the latest, and since the change shouldn't be breaking, just keep my self on master as well.

I feel an example shouldn't have not stable libraries cause it could confuse developer in their learning process.

I'm gonna take look at what you point and will see if I can change the other dependencies while on it.